src/shared/fonts_utils.js

```diff
@@ -413,16 +413,16 @@ function writeToFile(aBytes, aFilePath) {
   }
 
   netscape.security.PrivilegeManager.enablePrivilege('UniversalXPConnect');
-  var Cc = Components.classes,
-      Ci = Components.interfaces;
-  var file = Cc['@mozilla.org/file/local;1'].createInstance(Ci.nsILocalFile);
+  const Cc = Components.classes,
+        Ci = Components.interfaces;
```

Comment:
I think we should simply make a second statement for this since we aim to avoid multiple declarations in one statement as much as possible for readability (we barely have it in the codebase, and this part is fairly old code):

```javascript
const Cc = Components.classes;
const Ci = Components.interfaces;
```

Reply:
I'd be happy to do it in another PR for clarity of our actions, what do you think?

Reply:
> I'd be happy to do it in another PR for clarity of our actions,

In this *particular* case, the changes are too small to warrant another PR in my opinion; hence simply addressing all comments and squashing everything together should suffice.

Reply:
Yes, please keep it within this PR and squash all the changes together.

Reply:
Done!